_example/version/version.go

```diff
@@ -9,22 +9,15 @@ import (
 )
 
 func New(c *gin.Context) (string, error) {
-	var header string
-
-	if len(c.Request.Header["Accept"]) > 0 {
```

Comment:
Hmm, this is workaround of `out of index` error, you wrote at #90.

Do you really want to replace this?